README.md

```diff
@@ -12,3 +12,4 @@
 | 2 | Веня Стаховский | Интерстеллар                           |
 | 3 | Михаил Антаманов| Питер-ФМ                               |
 | 4 | Ири Айларова    | Звездные войны                         |
+|100000 | Дима Зубков |    Матрица                             |
```

Comment:
The number is wrong. And the tabulation should be kept. Please fix.

Reply:
Sorry, how many more interesting new conditions will suddenly be introduced? You actually skipped my commit from las night, didn't give me any feedback, then today it turned out that a lot of other, newer commits got in.

What exactly am I supposed to do?

Reply:
All this achieved so far is making me feel like this is a waste of time.

Reply:
![image](https://user-images.githubusercontent.com/75221640/108924906-a8695700-764c-11eb-96ac-8b794d54393d.png)

Reply:
This also tells me that the tabulation works by parsing the symbols, not by counting spaces 
![image](https://user-images.githubusercontent.com/75221640/108925644-14988a80-764e-11eb-9b19-cd42559d17d1.png)

Reply:
I couldn't accept your pull request because you didn't act on feedback.